axelor-account/src/main/java/com/axelor/apps/account/db/repo/InvoiceManagementRepository.java

```diff
@@ -76,4 +78,22 @@ public Invoice save(Invoice invoice) {
       throw new PersistenceException(e.getLocalizedMessage());
     }
   }
+
+  @SuppressWarnings({"unchecked", "rawtypes"})
+  @Override
+  public Map<String, Object> populate(Map<String, Object> json, Map<String, Object> context) {
+    try {
+      if (context.get("_model") != null
+          && context.get("_model").toString().contains("SubrogationRelease")) {
+        long id = (long) context.get("id");
+        SubrogationRelease subrogationRelease =
+            Beans.get(SubrogationReleaseRepository.class).find(id);
+        json.put("$SubrogationStatusSelect", subrogationRelease.getStatusSelect());
```

Comment:
I'm not sure if this can cause problems, but maybe it would be safer to not start a context variable name with a uppercase letter.